### ISSUE TITLE
Make Vox exclusive to Trader and Vox Raiders, develop the Outpost to host at least six

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#10613]
This is a discussion issue ticket since I have been asked for one, here is the proposed plan that ought to be discussed first before server poll and then implementation if validated by playerbase : 
- Vox characters now have access to one job only : Trader. They are auto-signed up for it. They can also opt in for Vox Raiders should it happen
- Due to the maximum slots, it might be wise to have some catch-all in case some Voxes can't sign up for Trader job. A possibility is whitelisting Vox and making them build up a human, and finding another way for them to name and design a Vox they might play
- The Outpost on all maps is to be significantly extended to fit this, something I can handle fine. Things like a basic Security wing for Miner shitler break-ins, basic Medbay treatment room, an ore smelter and a small R&D room for trading with benefit
- A modification to Space Law will be added barring Vox from having any station job and accessing non-Civilian areas, with a single exception in the form of Medbay in case of medical emergencies. Failure to comply will lead to an automatic holding waiting for deportation back to Outpost, or/and stripping of any way to access restricted area and restraining order

This is a discussion ticket, so now it's up to the people below

**Comment by kol1th** [comment]
You think we have the population for this?

**Comment by Konater** [comment]
Only if you code snekpeople

**Comment by JustSumBody** [comment]
Why?

**Comment by ghost** [comment]
Can't help but feel this'd become an all in or bust scenario, either all species are approached this way regarding limiting their ways of spawning, or we keep it as is.

Doesn't matter if we do it all at the same time or across a long period of time, if we start a path, we're gonna get stuck with it.

Also, this'd make stealth raiders a whole lot more 'interesting' if they can't get contacts on station. Hell, even the vox traders would probably get the boot as soon as they dock because they're 'valid under space law'.

**Comment by ghost** [comment]
gr8 idea, wish I could upvote even more

But as kernikov says below, traders and raiders both have very little going on. This idea would be perfect were more things added to make these roles NOT SHIT

**Comment by kernikov** [comment]
Wew lad, just what we need.
Another half-assed vox insert.
Raiders is still shit, there's literally nothing you can do if they decide to trade unless you go full xenophobe gas them all. Something that isn't approved by space law.
Trader is also still shit, someone who doesn't have a role on the station. They exist to what, make petty trades? Best part, they can get antagonist roles like vampire. Seriously.

**Comment by Intigracy** [comment]
Stop trying to fix what isn't broken.

Whitelisting is a fucking horrible idea.

**Comment by Gringonius** [comment]
At this point I think it would be a net gain for the server to have all non-humans except maybe botany-grown dionas whitelisted.

The drama just keeps building up. I think it's time to let go and admit that it was not a good idea for this place.

**Comment by Intigracy** [comment]
What drama?

**Comment by kol1th** [comment]
@Gringonius [citation needed]

**Comment by icantthinkofanameritenow** [comment]
@Intigracy 
Let's just say that Bay had the right idea in making Vox an antagonist-only race. Nonhumans get nothing but special privileges these days- their own not-an-assistant with jetpacks and EVA job, their own whole subset of botany, even their own non-slowing magboots. I'd be honestly surprised if anyone plays them for any reason other than powergaming.

Doesn't help that the main advantage for humans (ASIMOV protection) is commonly made irrelevant by how quickly an AI's laws get changed to Robocop or NT default.

**Comment by Probe1** [comment]
there is no emoji for shrugging so here is a camel :camel:

**Comment by 9600bauds** [comment]
> I'd be honestly surprised if anyone plays them for any reason other than powergaming.

People flip a shit over "goodbye sawbones" but this gets a pass desu baka senpai

**Comment by TheFriggenSawbones** [comment]
@icantthinkofanameritenow 
some people enjoy the race aesthetic and not the numbers that go with

**Comment by kol1th** [comment]
> some people enjoy the race aesthetic and not the numbers that go with

I play vox exactly for this, I like the suit and their overall looks, especially with the new-ish different colored vox

**Comment by Kurfursten** [comment]
To be honest I didn't really care about the race at all until Traders got brought in, but I actually do the job and try to trade fun stuff to crew. So this PR wouldn't really affect me. And I don't like Vox heads as much as the next guy.

But I know a lot of good Vox players who like to play engineering, cargo, and medical, and I think it'd be a shame for them to get punished just to emphasize traders more.

To quote the above, "Don't fix what isn't broken" - the best solution is to make traders more useful - provide them with goods to trade, NOT to take away all else

**Comment by cowboybuttsdrivemenuts** [comment]
Why are you coding this instead of fixing actual problems? It's nice to have a punching bag race on-station, plus their suits look neat.

**Comment by icantthinkofanameritenow** [comment]
@kol1th 
I'd be more inclined to believe you if that aesthetic wasn't hideous in every possible way. They look even more like an alien monster from a B-movie than the greys somehow, and at least they can RP without sounding functionally retarded.  

@Kurfursten

> But I know a lot of good Vox players who like to play engineering, cargo, and medical, and I think it'd be a shame for them to get punished just to emphasize traders more.

Since when is making people play another race "punishment"? They aren't being fucking banned. 

As I said, their aesthetic is horrifically ugly and nobody can actually RP a nonhuman well enough to justify their playing one. Speaking of "RPing", that's all Trader is- why bother bartering for items that are either not worth the effort to trade for or outright useless? If you gave them access to things that were actually good like swiss army knives or switchtools, that might be a start. 

As an alternative though, I suggest simply whitelisting nonhuman races instead. The precedent already exists for skellingtons, and nobody went this far out of their way to defend them when they were in this position.

**Comment by TheFriggenSawbones** [comment]
why do you want to take away an innocuous race option and please be careful citing rp on the light rp server.

**Comment by Intigracy** [comment]
Skeletons were removed because they were originally added as a halloween thing and were insane meme machines.

Plasmamen exist as an alternative to skeletons that make more sense in a space setting.

I highly disagree with removing all nonhumans from play, which is in essence what will happen if a whitelist is introduced.

Whitelists are fucking retarded.

**Comment by kol1th** [comment]
@icantthinkofanameritenow 
I enjoy the aesthetic of the whole double jointed knees thing, and the suits (especially chemist/med), what makes your opinion any more important than any other? Also, I make a point of not sounding like "I AM VOX NO ENGLISH GOOD SKREE SKRAW". I fuck up the English occasionally, but not to the point that it makes me look retarded. Also, never in Pidgin, that's just retarded.

Also, this is all based off of your definitions now. Also, people are definitely suggesting more things for them to trade, and Kurfurst is quite literally coding new things for them to trade, that actually makes trading valued by having things not normally obtainable.

Also, as sawbones said, >RP on a light RP server
For what purpose? Is that why you think they should be removed? You don't like how they look and people can't RP them how you think is correct? Or that only powergamers play Vox?

**Comment by TheFriggenSawbones** [comment]
some people just want to be dinosaur freaks whats the problem with that

**Comment by icantthinkofanameritenow** [comment]
@TheFriggenSawbones 
For the same reason I might take away a fifth wheel on a car- it may not be doing harm, but it sure is a waste of space unless you can explain how the car benefits from having it. Sure, Kurfurst is adding "stuff", but I don't see how a few functionally identical plants could possibly be worth trading for. 

@kol1th 
They have as much merit as yours or anyone else's, and this is a discussion. But if you want me to contribute, I'll say that a few plants alone won't be enough. They need one-of-a-kind gadgets that can't be made on the protolathe, or tools that are objectively superior to the station's equivalents. Something where the station might actually want to barter for them, as opposed to shooting the trader and taking the goods off his bloody corpse.

**Comment by Kurfursten** [comment]
Looks like @icantthinkofanameritenow is beginning his removal crusade. First the Traders, next the Librarians, the Bartenders, the Janitors. Each feature in the game from tearing bedsheets into rags to xenoarcheology finds will have to prove their worth to him or be removed.

**Comment by Durgen** [comment]
I play vox exclusively for trader so I wouldn't mind making them trader only at all.

> Whitelisting 

Shitty Idea. I don't want to be the asshat playing a whitelisted race.

> Space Law moficiatoin

Sounds extreme and unnecessary. Traders are already barred from most departemnts by the HOP. It should be their perogative though.
If there are no engineers/medbay/whatever and a trader wants to join the crew they should be allowed to fill in as needed.

edit: before touching vox, plasmamen need to be dealth with. [spoiler]sorry sawbones[/spoiler]

**Comment by TheFriggenSawbones** [comment]
@icantthinkofanameritenow 
So arbitrarily removing player choice is a good thing?

**Comment by icantthinkofanameritenow** [comment]
@Kurfursten 
Oh no, that's not true. It's just right now Traders are little better than Assistants with a gimmick, and I would be equally irritated if it was (for example) greys that were getting special treatment over other races. 

#10616 is a good start as to what I would prefer. If Vox get greytide 2.0 as their special job, then humans should be the only ones who should play Heads. Plus, if you're going to introduce a job that isn't purely RP, then you should have something to back it up. IAAs have their faxes, Janitors have their cleaning abilities and in extension keeping the serb from lagging more than usual, and even Librarians and Clowns have earned their place in SS13 culture. Traders on the other hand have only some simple goods that are barely worth the effort of interacting with them to get. Plus, you only need one to completely make them irrelevant. 

Now if those seeds could _only_ be extracted by Vox seed extractors that would be a step forward. But that isn't the case.

**Comment by JustSumBody** [comment]
You're inventing drama that just doesn't exist.
The issues with the Vox Raiders mode would exist even if you replaced Vox with humans.
The Vox Outpost is a relatively new thing, finding/making new and unique things to add to it is in progress. 
Any complaints about them being used for powergaming could easily be applied to other nonhuman races.
Also 

> nobody can actually RP a nonhuman well enough to justify their playing one. Speaking of "RPing", that's all Trader is- why bother bartering for items that are either not worth the effort to trade for or outright useless?

You're arguing nobody can RP them well, then arguing that there's no point in the job since it's dedicated purely to RP trading. Then there was your comment in OOC about them not fitting "in the lore". All this on a low RP server to boot. I'm just not understanding your intent.

> and even Librarians and Clowns have earned their place in SS13 culture

These are also jobs that have been in the game for many years. Vox Traders were merged 2 months ago. I say give it time.

**Comment by Kurfursten** [comment]
"Librarians are more a part of Space Station 13 culture than GAS THE VOX and bird jews" - I'm not even going to engage if you're just going to post joke replies.

**Comment by icantthinkofanameritenow** [comment]
My intent is admittedly "nonhumans get too many special privileges". Having an entire job be exclusive to one nonhuman race just screams of bias to me. It did when it was first proposed, and still does now.

Talk all you like about me limiting options, but it's hypocritical of you to call me out on that when the race designated as the punching bag gets more options than anyone else does. Favoritism much?

**Comment by TheFriggenSawbones** [comment]
and removing vox from every job but trader is not bias?

Like we're playing a game here how is removing Vox fun, for like anyone.

Why not code a special job for humans if it bothers you that much, no one would mind.

**Comment by Kammerjunk** [comment]
We have some of those.
They're the Chief Medical Officer, Research Director, Chief Engineer, Head of Security, Head of Personnel, and Captain.
:^)

**Comment by Kurfursten** [comment]
@icantthinkofanameritenow You know I just want you to know that even though tempers flared with this whole issue, I understand where you're coming from. I played human exclusively for 2 years. I just like the Trader job and that's the only reason I've ever even played Vox.

@Kammerjunk You know, I've been meditating on this in the other thread and I realized I'd prefer it if instead of head roles, it was "all roles with a loyalty implant must be human". So all of sec, IAA, HOP, and Captain. Makes sense, right? I mean, those are the NT loyalists on the station so of course they're human. You might bring in a weirdo grey to be the research director, but who wants to trust Vox with the armory as Warden?

**Comment by Gringonius** [comment]
@Intrigracy @kol1th Issues and complaints about balance between races, endless whining about players of certain races from people that take the game too seriously, a shitload of Asimov discussion cut short with non-humans being a minority.
I guess it would be an easy way out. To me "whitelisted" on /vg/ is actually removed from play.

**Comment by kol1th** [comment]
@Gringonius I'm not sure what you're trying to make a point of here.

**Comment by Intigracy** [comment]
That's what he's citing as drama.

**Comment by Orangerush** [comment]
Only after you make plasmamen spawn in a roid dumpster

**Comment by Praecordia** [comment]
antag-only vox when?

**Comment by icantthinkofanameritenow** [comment]
@Praecordia 
Not soon enough.

**Comment by ghost** [comment]
If we're going to go around restricting races from certain jobs, then we have to do it across the whole board. The vox trader job isn't even good, you're mostly ignored and by the time you have anything worthy of trading, no one will want it because the station will be in shambles due to antags or the likes. There's a reason the number of people playing it dropped two weeks after it came out. If you're gonna restrict vox to raider and trader, you might as well remove both. Shit job, shit gamemode. But people will then complain even more because they can't play vox anymore. I don't even understand the arguements for this. Something about furries? Do you realise we're a 4chan server, where 90% of our playerbase are degenerates with incredibly varied likes and dislikes. Just by playing as a bird-race doesn't mean you're enabling furries or anything, neither would if we had round-start catbeasts. Erotic and disturbing behavior is bannable, If these people caused a problem, it would be dealt with. Except no one does. People are just picking on vox because they have a bad reputation due to some of the players. Are you trying to say humans are somewhat better? There are plenty of TERRIBLE players who exclusively play human. As there are the same who play Grey, Plasmaman and Vox.
